swift/ios-shoppe-demo/Views/ThankYouCell.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ThankYouCell.swift
+//  ios-shoppe-demo
+//
+//  Created on 5/6/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+import UIKit
+
+class ThankYouCell: UITableViewCell {
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
```

Comment:
Does this comment apply to the line above?  Can we move it above or on the same line if so?  It looks like there's a comment and then some code was removed.